source/dub/commandline.d

```diff
@@ -1239,10 +1243,10 @@ class FetchCommand : FetchRemoveCommand {
 		else {
 			try {
 				dub.fetch(name, Dependency(">=0.0.0"), location, fetchOpts);
-				logInfo(
-					"Please note that you need to use `dub run <pkgname>` " ~
-					"or add it to dependencies of your package to actually use/run it. " ~
-					"dub does not do actual installation of packages outside of its own ecosystem.");
+				logInfo("Finished", Color.green, "%s fetched", name.color(Mode.bold));
+				logInfo("Please note that you need to use `dub run <pkgname>` or add it");
+				logInfo("to dependencies of your package to actually use/run it.");
+				logInfo("Dub does not do actual installation of packages outside of its own ecosystem.");
```

Comment:
I don't know if it's a good idea to separate the lines, if the window is small it will wrap each one individually

Reply:
You are right. I think I made that change because now, since logInfo("...") lines are indented by 12 chars (or whatever is configured as tag width in the logging module), if the terminal automatically wraps them, they look a bit messier since the wrapped part doesn't obviously get indented as well.
I don't have any problem restoring that text to how it was before if, in your opinion, that's a better idea. Let me know.

Reply:
maybe the log function could wrap with indentation based on the terminal width, though that would be a much more complex function